### PR TITLE
Migrated test_glusterd_split_brain

### PR DESCRIPTION
**Pull request by nik-redhat**
Migrated TC [test_glusterd_split_brain](https://github.com/gluster/glusto-tests/blob/master/tests/functional/glusterd/test_glusterd_split_brain.py)

Fixes: #588 
Updates: #292 
Signed-off-by: nik-redhat <nladha@redhat.com>

### CODE REVIEW

**Comment by nik-redhat** [comment]
This TC requires 6 servers to run, hence not attaching the ouput.

**Comment by srijan-sivakumar** [comment]
> This TC requires 6 servers to run, hence not attaching the ouput.

Now here's what we'll do...in the demo we will just run these tests :p all tests which need 6 servers 😆

**Comment by nik-redhat** [comment]
> > This TC requires 6 servers to run, hence not attaching the ouput.
> 
> Now here's what we'll do...in the demo we will just run these tests :p all tests which need 6 servers

Mtlb sab barbaad :stuck_out_tongue_winking_eye:

**Comment by srijan-sivakumar** [comment]
> > > This TC requires 6 servers to run, hence not attaching the ouput.
> > 
> > 
> > Now here's what we'll do...in the demo we will just run these tests :p all tests which need 6 servers
> 
> Mtlb sab barbaad stuck_out_tongue_winking_eye

psst..psst..Not so loud :p But still maybe something similar to gutihub.com for curious souls.

File: tests/functional/glusterd/test_glusterd_split_brain.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+ 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+ Description:
+   TC to check split brain in glusterd with quorum set
+"""
+
+import traceback
+from tests.nd_parent_test import NdParentTest
+
+# nonDisruptive;
```
Comment (srijan-sivakumar) [inline tests/functional/glusterd/test_glusterd_split_brain.py:25]: disruptive as we are stopping glusterd.

**Review by srijan-sivakumar — CHANGES_REQUESTED** [review]

File: tests/functional/glusterd/test_glusterd_split_brain.py (hunk 1 above)
Comment (nik-redhat) [reply tests/functional/glusterd/test_glusterd_split_brain.py:25]: :grin:

**Review by srijan-sivakumar — APPROVED** [review]

**Pull request merged**